2019/09/27 14:03:31 [error] 41439#41439: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 14:07:11 [error] 41439#41439: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 14:30:37 [error] 41439#41439: *7 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 14:39:50 [crit] 2599#2599: *1 connect() to unix:/var/run/php5-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "backend.test"
2019/09/27 14:44:35 [crit] 2599#2599: *3 connect() to unix:/var/run/php5-fpm.sock failed (2: No such file or directory) while connecting to upstream, client: 127.0.0.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://unix:/var/run/php5-fpm.sock:", host: "backend.test"
2019/09/27 15:37:20 [error] 3475#3475: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 16:58:44 [error] 48153#48153: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:16:58 [error] 48437#48437: *6 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:20:04 [error] 48437#48437: *8 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:20:28 [error] 48481#48481: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:20:31 [error] 48481#48481: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:20:39 [error] 48481#48481: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:23:39 [error] 48545#48545: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:23:59 [error] 48545#48545: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:25:31 [error] 48584#48584: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:25:47 [error] 48584#48584: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:26:09 [error] 48584#48584: *3 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:33:27 [error] 48741#48741: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:34:58 [error] 48741#48741: *5 connect() failed (111: Connection refused) while connecting to upstream, client: 192.168.1.10, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
2019/09/27 17:35:50 [error] 48741#48741: *11 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: backend.test, request: "GET /advanced/frontend/web/ HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "127.0.0.1"
2019/09/27 17:35:58 [error] 48741#48741: *13 connect() failed (111: Connection refused) while connecting to upstream, client: 127.0.0.1, server: backend.test, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "backend.test"
